CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
-## 2.2.1 - TBD
+## 3.0.1 - TBD
```

Comment:
As noted in the zend-expressive-router 2.0.0 PR, remove this stub, and re-add it _after_ tagging 3.0.0.